### PR TITLE
133: Update localized text for SettingView

### LINKED ISSUE #133: Update Copy on Settings View
**Describe the solution you'd like**
* Update the copy on the `SettingsView`, be sure to update the German localization.
* See the `Localization` files: `BrainMarksIsOpenSource...` and `IfYouWantToContribute`
* Also please change the key `BrainMarksIsOpenSource...` to not have 3 dots

* First Block
```
Brain Marks is an open source project! It was created during the Big Brain Hackathon and has continued to be open sourced for Hacktoberfest.
You can categories your saved tweets.
```
* Second Block
```
You can contribute to this project!
This project is open source on GitHub. Visit the link below to contribute. We have a list of all contributors, your name could be there!
```

**Describe alternatives you've considered**

**Additional context**
* Current copy in the first block
Screen Shot 2022-10-01 at 6 35 55 PM

* Current copy in the second block
Screen Shot 2022-10-01 at 6 34 16 PM

### PR DESCRIPTION
**Pull request by Drag0ndust**
Progress / Closed on issue #

## What it Does
* Updated the localised files for the SettingsView

## How I Tested
* Start the App and tab on Settings
* Check the updated text

Closes #133

### CODE REVIEW

File: brain-marks/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
 // SettingsView
 "Settings" = "Settings";
-"BrainMarksIsOpenSource..." = "Brain-marks is an open source project! It was created for the Big Brain Hackathon and during Hacktoberfest 2021, work has continued.";
+"BrainMarksIsOpenSource" = "Brain-marks is an open source project! It was created for the Big Brain Hackathon and during Hacktoberfest 2021 and 2022, work has continued.";
```
Comment (mikaelacaron) [inline brain-marks/Localization/en.lproj/Localizable.strings]: Can you use the exact text that I noted in #133 ? I didn't want to include the year anymore, and the hyphen in "brain marks" is gone at the beginning

File: brain-marks/Localization/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -42,7 +42,7 @@
...
 "IfYouWantToContribute" = "If you are interested in contributing yourself, visit the GitHub Repo for more information. Below a list of all contributors can be found. \nYour name could be there!";
```
Comment (mikaelacaron) [inline brain-marks/Localization/en.lproj/Localizable.strings]: Can you also use the exact text I changed in #133?

**Review by mikaelacaron — CHANGES_REQUESTED** [review]

File: brain-marks/Localization/en.lproj/Localizable.strings (hunk 1 above)
Comment (Drag0ndust) [reply brain-marks/Localization/en.lproj/Localizable.strings]: Changed it accordingly to the issue

File: brain-marks/Localization/en.lproj/Localizable.strings (hunk 2 above)
Comment (Drag0ndust) [reply brain-marks/Localization/en.lproj/Localizable.strings]: Changed it accordingly to the issue

**Review by mikaelacaron — APPROVED** [review]

**Pull request merged**